tests/scripts/thread-cert/Cert_5_1_07_MaxChildCount.py

```diff
@@ -40,7 +40,7 @@ class Cert_5_1_07_MaxChildCount(unittest.TestCase):
     def setUp(self):
         self.nodes = {}
         for i in range(1, 13):
-            self.nodes[i] = node.Node(i)
+            self.nodes[i] = node.Node(i, (i >= 3))
```

Comment:
prefer using `MTDS` list as the other cases.

Reply:
Unlike other test cases, ED and SED devices are not named in this test case. I believe we could refine this after they are named.